retention/README.md

```diff
@@ -0,0 +1,8 @@
+# To run tests locally
+Create the local postgres instance.
+
+```$ docker run --name postgresql-container -p 5432:5432 -e POSTGRES_USER=test -e POSTGRES_PASSWORD=somePassword -d postgres```
+
+Run tests with the appropriate environment variables.
+
+```SYNAPSE_DB_HOST=localhost SYNAPSE_DB_USERNAME=test SYNAPSE_DB_PASSWORD=somePassword SYNAPSE_DB_DATABASE=postgres SYNAPSE_DB_OPTIONS= STATS_DB_HOST= STATS_DB_USERNAME= STATS_DB_PASSWORD= STATS_DB_DATABASE= python3 test/test_cohort_analysis.py```
```

Comment:
I'm having trouble running this locally. Python is searching for the `cohort_analysis` file at `.`, which is not the case since `test_cohort_analysis.py` in the `test` directory.

```
> SYNAPSE_DB_HOST=localhost SYNAPSE_DB_USERNAME=test SYNAPSE_DB_PASSWORD=somePassword SYNAPSE_DB_DATABASE=postgres SYNAPSE_DB_OPTIONS= STATS_DB_HOST= STATS_DB_USERNAME= STATS_DB_PASSWORD= STATS_DB_DATABASE= python3 test/test_cohort_analysis.py
Traceback (most recent call last):
  File "/home/user/code/stats/retention/test/test_cohort_analysis.py", line 3, in <module>
    import cohort_analysis as ca
ModuleNotFoundError: No module named 'cohort_analysis'
> 
```

I'm actually not sure how easy it is to rectify this without converting `retention` into a Python module.

Reply:
In development I was running this from the retention directory, so it was able to find cohort_analysis.py, I've updated the README to make it easier to run from the root directory.